samples/cpp/benchmark_app/main.cpp

```diff
@@ -600,6 +600,11 @@ int main(int argc, char* argv[]) {
 
         // If set batch size, disable the auto batching
         if (FLAGS_b > 0) {
+            if ((!FLAGS_hint.empty() && (FLAGS_hint == "throughput" || FLAGS_hint == "tput" ||
+                                         FLAGS_hint == "cumulative_throughput" || FLAGS_hint == "ctput")) ||
+                (FLAGS_hint.empty() && FLAGS_d.find("MULTI") != std::string::npos)) {
```

Comment:
Does MULTI without hint activate auto batching? I can't see it being mentioned in https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/automatic_batching.md#enablingdisabling-automatic-batching

Also this page claims that auto batching activates only for GPU. Should this condition be added to your implementation?

Reply:
The warning of claims the behavior of "-b 1". Then we can get same warning message on below scenarios:

-  -d CPU -b 1
- -d GPU -b 1
- -d AUTO -b 1
- -d MULTI:GPU,CPU -b 1